source/Octopus.Tentacle.Tests.Integration/ClientScriptExecutionRetriesTimeout.cs

```diff
@@ -43,9 +47,9 @@ public async Task WhenRpcRetriesTimeOut_DuringGetCapabilities_TheRpcCallIsCancel
                     .DecorateCapabilitiesServiceV2With(d => d
                         .BeforeGetCapabilities((service) =>
                         {
-                            service.EnsureTentacleIsConnectedToServer(Logger);
```

Comment:
As responses are cached, we can't make a call to the ICapabilitesService to ensure tentacle is connected but need to use another service.